.goreleaser.yaml

```diff
@@ -61,3 +91,37 @@ docker_manifests:
   image_templates:
   - ghcr.io/pfnet/{{ .ProjectName }}:{{ .Version }}-amd64
   - ghcr.io/pfnet/{{ .ProjectName }}:{{ .Version }}-arm64v8
+
+krews:
+- name: hns
+  ids: # archives IDs
+  - kubectl-hns
+  short_description: Manage hierarchical namespaces (part of HNC)
+  skip_upload: true
```

Comment:
This setting appears to skip uploading to the krew-index repo rather than skipping uploading as release artifacts. Since it is acceptable to upload as release artifacts, it seems that this setting can be deleted.

```suggestion
```